src/OrbitGl/DataManager.h

```diff
@@ -81,6 +81,13 @@ class DataManager final {
   }
   [[nodiscard]] bool get_enable_introspection() const { return enable_introspection_; }
 
+  void set_enable_user_space_instrumentation(bool enable) {
+    enable_user_space_instrumentation_ = enable;
+  }
+  [[nodiscard]] bool get_enable_user_space_instrumentation() const {
```

Comment:
I know that the one above is called `get_enable_introspection`, but that's an anomaly, when we use `snake_case` for getters we omit the `get` (so `set_property` for the setter, `property` for the getter), such as for `collect_thread_states` above and `samples_per_second` below.

Reply:
Thanks! Removed the `get_`.